### ISSUE TITLE
Invisible buttons in OBS Studio's UI when using the high-contrast black theme in Windows 10

### ISSUE THREAD

**Issue by k-kolev1985** [obsproject/obs-studio#3362]
### Description
Another accessibility issue report. In some windows of OBS Studio, there are some buttons, which are not visible, if the theme used in Windows is the high-contrast black one.

### Steps to reproduce it:
1. Enable high-contrast mode in Windows 10, by doing the following:
1.1. Open Windows Settings (Windows+I).
1.2. Go to Ease of access -> High-contrast.
1.3. Turn on the high-contrast mode, using the respective toggle.
1.4. From the "High-contrast theme" combo-box, select "High-Contrast Black".
2. Launch OBS Studio.
3. Create a scene and add at least one source to it.
4. Observe the small rectangular buttons in the main window of OBS Studio and other windows of the program (e.g. for audio/video filters, Settings -> Hotkeys, and probably others that I don't know of or can't remember at the moment).

### Expected result:
The buttons should have a visible rectangle around them, as do the buttons on the bottom right of the main Window of OBS Studio (for starting recording, streaming, exiting the program, etc.) and their icons should be of a color that does not conflict with the background color of the buttons themselvs.

### Actual result.
The buttons don't have a visible rectangle around them.) Their icons are black. Both things make those buttons invisible for users who use the high-contrast black theme of Windows.

### Test environment
- Operating system: Windows 10 Pro version 2004 (build 19041.450), 64-bit, in Bulgarian with all locale settings set to "Bulgarian".
- OBS Studio version: 25.0.8 (64 bit), portable, in english.
- Processor: Intel Core i5-9400F at 2.90GHz.
- RAM Memory: 16.00GB.
- Graphics: MSI GeForce GTX-1050TI Gaming X 4G, 4096MB dedicated memory, desktop resolution set to 1920x1080.
- Sound Card: Realtek ALC892 at Intel Cannon Point PCH.

**Comment by WizardCM** [comment]
OBS Dark Theme is mostly usable (a few buttons change colour), but the System theme definitely becomes unusable. This'd be because we essentially hardcode dark icons.

**Comment by Fenrirthviti** [comment]
Closing as stale. If this is still an issue please open a new issue with updated reproduction steps on the latest version of OBS.

The affected themes are no longer available in OBS anyway.
